build-scripts/webpack.js

```diff
@@ -91,7 +91,7 @@ const createWebpackConfig = ({
       ),
     ].filter(Boolean),
     resolve: {
-      extensions: [".ts", ".js", ".json", ".tsx"],
+      extensions: [".ts", ".js", ".json"],
```

Comment:
Epic 🎉 

Guess we might need to add it back once we start working with the calendar ?